lib/main.js

```diff
@@ -2,7 +2,7 @@ const URL = require('url');
 const UTIL = require('./util.js');
 const QS = require('querystring');
 
-const nextpagRegex = /<div class="(([^"]*branded\-page\-box[^"]*search-pager[^"]*)|([^"]*search-pager[^"]*branded\-page\-box[^"]*))">/; // eslint-disable-line no-useless-escape, max-len
+const nextpagRegex = /<div class="(([^"]*branded\-page\-box[^"]*search\-pager)|([^"]*search\-pager[^"]*branded\-page\-box)).*">/; // eslint-disable-line no-useless-escape, max-len
```

Comment:
why not remove it all together?
```suggestion
const nextpagRegex = /<div class="(([^"]*branded\-page\-box[^"]*search\-pager)|([^"]*search\-pager[^"]*branded\-page\-box))/; // eslint-disable-line no-useless-escape, max-len
```

Reply:
Seems reasonable assuming it's not later parsing data between brackets those.

Reply:
Seems reasonable assuming it's not later parsing data between brackets those.